client/lib/service-worker/service-worker.js

```diff
@@ -104,5 +104,30 @@ self.addEventListener( 'message', function( event ) {
 /* eslint-disable */
 self.addEventListener( 'fetch', function( event ) {
 	// this listener is required for "Add to Home Screen" support
+	const request = event.request;
```

Comment:
Is this required to have an MVP of a PWA? I wonder, because we specifically moved all the fetching/caching logic to a new PR https://github.com/Automattic/wp-calypso/pull/26666 which implements an offline page.

Reply:
If you run a PWA audit through Chrome without these changes it will error.
![image](https://user-images.githubusercontent.com/2810519/44353599-56805e80-a45c-11e8-8e6d-95771f10ac01.png)
My read of https://developers.google.com/web/tools/lighthouse/audits/install-prompt is that the fetch event handler has to actually fetch or cache the `start_url` for it to prompt.

Reply:
How about regardless let's just narrow the scope of this PR to the changes to the Manifest so we can move on? My changes are just going to go away with the next PR anyway

Reply:
Yeah agreed, I feel this PR is complex enough as it is 👍